spark/src/test/scala/org/apache/spark/sql/benchmark/CometTPCDSMicroBenchmark.scala

```diff
@@ -53,23 +53,24 @@ import org.apache.comet.CometConf
 object CometTPCDSMicroBenchmark extends CometTPCQueryBenchmarkBase {
 
   val queries: Seq[String] = Seq(
-    "add_many_decimals",
-    "add_many_integers",
-    "agg_high_cardinality",
-    "agg_low_cardinality",
-    "agg_sum_decimals_no_grouping",
-    "agg_sum_integers_no_grouping",
+//    "add_many_decimals",
```

Comment:
Did you want to keep these cases commented out?

Reply:
No. Thanks for catching that.